validator/src/bootstrap.rs

```diff
@@ -490,9 +490,9 @@ fn get_vetted_rpc_nodes(
             Ok(rpc_node_details) => rpc_node_details,
             Err(err) => {
                 error!(
-                    "Failed to get RPC nodes: {err}. Consider checking system clock, removing \
-                     `--no-port-check`, or adjusting `--known-validator ...` arguments as \
-                     applicable"
+                    "Failed to get RPC nodes: {err}. \
+                    Consider checking system clock, removing `--no-port-check`, \
+                    or adjusting `--known-validator ...` arguments as applicable"
```

Comment:
To Brooks' comment about minimizing the number of special cases handled in the future, I think just keep the `fmt` recommendation here.